policies/travel-policy.md

```diff
@@ -0,0 +1,352 @@
+    under entertainment, you can not then claim a dinner under your meal
+    entitlement).
+-   Alcoholic beverages are covered up to a maximum total cost of US\$20
+    per person per meal.
+
+### Other Expenses
+
+In addition, OpenSSL will reimburse the cost of:
+
+-   meeting, boardroom, or conference room &/or other facilities needed
+    while traveling for OpenSSL business purposes.
+-   Conference &/or attendance fees
+
+Personal Travel
+---------------
+
+OpenSSL will not be financially responsible for any personal expenses
+and travel arrangements.
+
+-   If a Traveller chooses to travel with a partner; their partner\'s
+    accommodation, meals, leisure, and/or travel expenses will not be
+    reimbursed.
+-   If a Traveller chooses to have additional days either prior or post
+    the required travel, costs associated with the additional days will
+    not be reimbursed, (excluding any pre/post days included in The
+    Policy)
+
+Reimbursements
+--------------
+
+The Traveller must keep all receipts, tickets etc related to the
+reimbursement and submit these with the **Expenses Reimbursement Form**.
+Photos or scanned versions of the receipts or credit card statements are
+acceptable. In the event a receipt is not available or lost the
+Traveller must submit a signed statement detailing the item/s purchased
+and the cost.
+
+The Traveller is expected to submit an **Expenses Reimbursement Form**
+within 30 days from the end of the trip.
+
+-   On the Travel Expense Reimbursement Request, the Traveller must
+    include details and photos or scanned receipts for all expenses with
+    an individual value greater than US\$30.
+-   For expenses under US\$30 receipts do not have to be provided
+    however details of the cost should be included on the **Expenses
+    Reimbursement Form**.
+-   The Traveller should cross out items unrelated to their
+    reimbursement claim on the receipts; ensure any entertainment
+    clearly lists the attendees and relevant company details; and
+    exclude any personal expenses.
+
+The Travel Authoriser or their designated representative is responsible
+for reviewing the Travel Expense Reimbursement Request to ensure it is
+in line with The Policy and the Travel that was approved.
+
+### Travel Expense Reimbursement Claim
+
+To submit a reimbursement claim for business related travel expenses:
+
+1.  Download & Complete the **Expenses Reimbursement Form**
+2.  Ensure receipts/documentation is clearly labelled with relevant
+    details (e.g. where sharing meals, crossing out personal expenses
+    etc).
+3.  Ensure you have entered the correct bank details for your
+    reimbursement to be paid into.
+4.  Scan or photograph all applicable receipts etc and attach them to
+    the **Expenses Reimbursement Form **email.
+5.  Submit the form & scanned/photographed receipts within 30 days to
+    finance@openssl.org attention to the Travel Approver for review.
```

Comment:
Here it says travel approver....but above we said travel authoriser. Does it have to be the same person that originally approved the travel, or is it different?